middleware/recoverer.go

```diff
@@ -40,12 +41,14 @@ func Recoverer(next http.Handler) http.Handler {
 	return http.HandlerFunc(fn)
 }
 
+var recovererErrorWriter io.Writer = os.Stderr
```

Comment:
can you leave comment above this stating why you added this pkg-level variable? (as it appears for ability to test the PrintPrettyStack function, but its helpful for other reasons to know its rationale.)

Reply:
@pkieltyka Done